### PR TITLE
Add scripts

### PR DESCRIPTION
**Pull request by lunaflight**
This will make the installation process easier and the user will not need to input arcane incantations to the terminal anymore

### CODE REVIEW

File: scripts/init.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+main ()
+{
+    pip install -r local-requirements.txt &&
+    python -m src.history.build
+}
+
+# Create virtual environment
+python -m venv venv
```
Comment (SpookyYomo) [inline scripts/init.sh:10]: venvs are managed in manners beyond the usual venv module, with python versioning and virtual environment managers such as [Poetry](https://python-poetry.org/), [PDM](https://github.com/pdm-project/pdm), [Pipenv](https://pipenv.pypa.io/en/latest/), [Anaconda](https://www.anaconda.com/), [Pyenv](https://github.com/pyenv/pyenv), [Virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/), [Hatch](https://hatch.pypa.io/latest/).
Nonetheless this is a good starting point for the less intense python developers of the project.

Comment (lunaflight) [reply scripts/init.sh:10]: added a note about it in `CONTRIBUTING.md`

**Comment by SpookyYomo** [comment]
In retrospect, bundling all these scripts in a singular Makefile might be more sensible.
You may refer to any other project who does a `make build` or similar command with a Makefile in the git root.

**Comment by lunaflight** [comment]
I've tried this before, and it got too complicated. I don't remember the exact reason why, but you are free to try using a Makefile or open an issue for this.

**Pull request merged**